server/middlewares/errors/notFound.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response, NextFunction } from 'express';
+import { CustomError } from '../../utils';
+
+const notFound = (error: CustomError, req: Request, res: Response, next: NextFunction) => {
+  if (error.code === 404) {
+    return res.status(404).send({
+      status: 404,
+      message: error.message,
+    });
+  }
+  return next(error);
+};
+
```

Comment:
The 404 handle error doesn't have 4 parameters, so it just takes req, res, and next parameters.
And this doesn't catch any errors so it doesn't have a code and a message, it just handles the endpoints that we don't handle in our project